CMakeLists.txt

```diff
@@ -27,18 +27,6 @@ list(APPEND CMAKE_MODULE_PATH
     ${CMAKE_CURRENT_SOURCE_DIR}/bldsys/cmake
     ${CMAKE_CURRENT_SOURCE_DIR}/bldsys/cmake/module)
 
-# Precompiled header macro. Parameters are source file list and filename for pch cpp file.
```

Comment:
We should probably clean up the precompiled header generation commands anyway.  However, I believe clang-tidy will work with PCH.